state/machine.go

```diff
@@ -1241,6 +1242,27 @@ func (m *Machine) InstanceId() (instance.Id, error) {
 	return instData.InstanceId, err
 }
 
+// Names returns both the provider's instance id and a user-friendly
+// display name. The display name is intended used for human input and
+// is ignored internally.
+func (m *Machine) InstanceNames() (instance.Id, string, error) {
+	instData, err := getInstanceData(m.st, m.Id())
+	if errors.IsNotFound(err) {
+		err = errors.NotProvisionedf("machine %v", m.Id())
+	}
+	if err != nil {
+		return "", "", err
+	}
+	var displayName string
+	if instData.DisplayName == nil {
+		displayName = ""
```

Comment:
No need for this - displayName is initialised as "". Just need the != nil case

Reply:
Resolved as we're no longer passing back a `*string`